lib/atomic_web/live/partner_live/show.ex

```diff
@@ -1,7 +1,7 @@
 defmodule AtomicWeb.PartnerLive.Show do
   use AtomicWeb, :live_view
 
-  import AtomicWeb.Components.Avatar
+  import AtomicWeb.Components.{Avatar, Button}
```

Comment:
No need to import `Button`, it's already imported globally
```suggestion
  import AtomicWeb.Components.Avatar
```